lecture00-introduction/introduction.tex

```diff
@@ -153,7 +153,9 @@ \section{Who we are}
 \begin{frame}{Who we are}
     Johannes Schmidt <johannes.schmidt@boku.ac.at>\\
     \begin{itemize}
-        \item
+        \item Associate Prof. in Energy \& Resource Economics at the Institute for Sustainable Economic Development
+        \item Works on modeling of renewable energy systems in R, GAMS (and python)
+        \item Has studied Computer Science at TU Wien
```

Comment:
I think we should agree on English grammar if possible, is it "Has studied" or "Studied" in this case?

Reply:
```suggestion
        \item Studied Computer Science at TU Wien
```